filesystem/gpt_partition_table.ksy

```diff
@@ -62,9 +66,11 @@ types:
         type: u4
       # We don't parse the trailing zeroed space
     instances:
+      sector_size:
+        value: 0x200
       entries:
         io: _root._io
-        pos: entries_start * 0x200
+        pos: entries_start * sector_size
```

Comment:
Just use `_root.sector_size` here, no need to declare it second time in `partition_entry`. As far as I understand, sector size is the same for whole disk.

Reply:
Much better that way